syncto/tests/test_functional.py

```diff
@@ -252,7 +259,7 @@ def test_collection_raises_with_invalid_sort_parameter(self):
         self.assertFormattedError(
             resp, 400, ERRORS.INVALID_PARAMETERS, "Invalid parameters",
             "_sort should be one of ('-last_modified', 'newest', "
-            "'-sortindex', 'index')")
+            "'-sortindex', 'index', 'last_modified', 'oldest')")
```

Comment:
Should this test also leave in the Sync client? (allow different sort keys in the client and not in the proxy)